website/docs/guides/collections.md

```diff
@@ -124,7 +124,7 @@ main(function*() {
 ```
 
 If we unpack this a bit, we can see that we're creating a new `Stream` called
-`textStream` using the method `map` on the stream. This stream gets the `text`
+`channel` using the method `map` on the stream. This stream gets the `text`
```

Comment:
I think what we want here is 

```
If we unpack this a bit, we can see that we're creating a new `Stream` called 
`textStream` using the method `map` on `channel`. This stream gets the `text`
property from each value in the stream
```

Reply:
Yes, that sounds correct. `textStream` is the channel we create, so the original correction is not quite correct, but maybe that is indicative of this sentence not being quite clear enough. @cowboyd's version reads a bit clearer IMO.

Reply:
And I used it ) I can't merge it,  because I don't have rights